dxw/advisories
# Prod release

## Description

Maintenance and branding updates.

## How to test

Compare:

* Staging: https://advisories.staging.dxw-govpress.dalmatian.dxw.net/
* Prod: https://advisories.dxw.com/

The differences should be:

* Cardo has been replaced by Poppins (see menu on top-right corner)
* The Twitter social media link has been replaced with Bluesky and links to a dxw account (see footer)

Behaviour should be the same on both desktop and mobile screen sizes.

## Screenshots

### Before

![Screenshot 2024-10-22 at 11 57 44](https://github.com/user-attachments/assets/89d17614-dd52-4f70-a134-0a6945c43d43)
...
![Screenshot 2024-10-22 at 11 57 55](https://github.com/user-attachments/assets/9f8989df-b8e8-48d9-94d2-63c29f4e8c1e)

### After

![Screenshot 2024-10-22 at 11 58 27](https://github.com/user-attachments/assets/3d301a4f-9abb-4314-8d8b-1391fa757a84)
...
![Screenshot 2024-10-22 at 11 58 35](https://github.com/user-attachments/assets/e5627d2e-61ca-4d4f-a399-9ddbda21afd2)
